websites/B/BoxSeries/presence.ts

```diff
@@ -0,0 +1,162 @@
+const presence = new Presence({
+		clientId: "955350055034945576"
+	}),
+	strings = presence.getStrings({
+		play: "presence.playback.playing",
+		pause: "presence.playback.paused",
+		browsing: "presence.activity.browsing"
+	}),
+	browsingTimestamp = Math.floor(Date.now() / 1000);
+
+let video = {
+	current: 0,
+	duration: 0,
+	paused: true
+};
+
+presence.on(
+	"iFrameData",
+	(data: { current: number; duration: number; paused: boolean }) => {
+		video = data;
+	}
+);
+
+presence.on("UpdateData", async () => {
+	const [Logo, Cover, Privacy, Buttons, Timestamps] = await Promise.all([
+			presence.getSetting<number>("logo"),
+			presence.getSetting<boolean>("cover"),
+			presence.getSetting<boolean>("privacy"),
+			presence.getSetting<boolean>("buttons"),
+			presence.getSetting<boolean>("timestamps")
+		]),
```

Comment:
We prefer the use of camelCase
```suggestion
	const [logo, cover, privacy, buttons, timestamps] = await Promise.all([
			presence.getSetting<number>("logo"),
			presence.getSetting<boolean>("cover"),
			presence.getSetting<boolean>("privacy"),
			presence.getSetting<boolean>("buttons"),
			presence.getSetting<boolean>("timestamps")
		]),
```